### PR TITLE
List structure project

### PR DESCRIPTION
**Pull request by Cathella**
- Used Webpack to bundle JavaScript.
- Used proper ES6 syntax.
- Created an empty To Do List placeholder in a template using the HTML Webpack Plugin.
- Created an index.js file and set an array of some simple to do tasks.
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load rendered the dynamically created list of tasks in the dedicated placeholder.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+// function to iterate over the tasks array and populate an HTML list item element for each task
+const iterateTasks = (task) => {
+  const li = document.createElement('li');
+  li.innerHTML = `
+    <div>
+      <input type="checkbox">
+      ${task.description}
+    </div>
+  `;
```
Comment (EricMbouwe) [inline src/app.js:28]: - [ ] You may have a `ul` tag here as a parent of your `li` elements to follow HTML best practices and tags hierarchy.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Cathella,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/app.js (hunk 1 above)
Comment (Cathella) [reply src/app.js:28]: @EricMbouwe, thank you for the review but the <ul> tag is already the parent tag in the template. These <li> tags are just children tags I am adding to the <ul>

Comment (EricMbouwe) [reply src/app.js:28]: Hi @Cathella,
Yeah, you are right, it is perfect! my bad, I didn't noticed that the parent was a `ul`, I thought that it was a div. Let's ask for another review and you will be good to go.

Comment (Cathella) [reply src/app.js:28]: Thank you @EricMbouwe, let me do that.

**Review by EricMbouwe — APPROVED** [review]
Hi @Cathella, 

# Status: Approved✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**